lib/reservation_monitor.py

```diff
@@ -99,10 +100,14 @@ def _check_flight_fares(self) -> None:
             try:
                 fare_checker.check_flight_price(flight)
             except RequestError as err:
+                if self.config.healthchecks_url is not None:
```

Comment:
Successful ping messages can also be sent inside of `schedule_reservations` on line 88 to let users know their reservations are being retrieved too.

Reply:
I'm not sure this is the best idea. Healthchecks is designed to have one URL per cron job. The "load reservations" cron job operates on a different schedule than the fare check cron job, and thus should have a separate healthchecks URL.

Reply:
`schedule_reservations` is always called right before the fare check. I understand the point about making it a separate URL though. If you think this would be useful to users as well, you are welcome to add a separate config for that too (it may be helpful if the script keeps failing to log in with a 429). If not, that can always be added later if people want it.

Reply:
However, looking at the current implementation, the success or fail notification is sent many times per interval. Would it be bad to add one additional healthcheck ping (I’m unfamiliar with Healthchecks and haven’t tried out this PR yet). Instead of in schedule_reservations, could there just be a failed/successful healthcheck sent for logging in (there is a try/except for logging in in the AccountMonitor?

Reply:
Hey @StevenMassaro, any thoughts about my comments above?

Reply:
Hey @jdholtz , sorry for the delay here. I think it would be best to add this as a separate config, in a separate PR.

Reply:
Yeah, I think we can wait to add it if someone wants the pings for checking flights. For now we can just leave it at fare checking.